website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-dashboard-status-tiles.md

```diff
@@ -52,3 +52,12 @@ Looker does not allow you to directly embed HTML, and instead requires creating
 - Once you have set up your custom visualization, you can use it on any dashboard! You can configure it with the exposure name, jobID, and token relevant to that dashboard.
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/dashboard-status-tiles/custom-looker.jpeg"/>
+
+### Tableau
+Tableau does not require you to embed an iFrame, instead you only need to use a Web Page object on your Tableau Dashboard, and use a URL in the following format.
```

Comment:
Tableau does not require you to embed an iFrame. You only need to use a Web Page object on your Tableau Dashboard and a URL in the following format.